scripts/updateUsers.ts

```
@@ -222,5 +247,4 @@ async function main() {
   }
 }
 
-// Run the script
-main().catch(console.error); 
+main().catch(console.error);
```

ESLint

[error] Insert `⏎`

(prettier/prettier)